webapp/static/interop-data.json

```diff
@@ -322,119 +322,118 @@
+                "mdn": "https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes",
+                "spec": "https://html.spec.whatwg.org/multipage/semantics-other.html#pseudo-classes",
+                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-pseudos",
                 "countsTowardScore": true
             },
             "interop-2023-property": {
                 "description": "Custom Properties",
-                "mdn": "",
-                "spec": "",
-                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-property", 
+                "mdn": "https://developer.mozilla.org/en-US/docs/Web/CSS/@property",
+                "spec": "https://drafts.css-houdini.org/css-properties-values-api/",
+                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-property",
                 "countsTowardScore": true
             },
             "interop-2023-flexbox": {
                 "description": "Flexbox",
-                "mdn": "",
-                "spec": "",
+                "mdn": "https://developer.mozilla.org/docs/Learn/CSS/CSS_layout/Flexbox",
+                "spec": "https://drafts.csswg.org/css-flexbox/",
                 "tests": "/results/css/css-flexbox?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2021-flexbox%20or%20label%3Ainterop-2023-flexbox", 
                 "countsTowardScore": true
             },
             "interop-2023-fonts": {
                 "description": "Font Feature Detection and Palettes",
-                "mdn": "",
-                "spec": "",
-                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-fonts", 
+                "mdn": "https://developer.mozilla.org/en-US/docs/Web/CSS/font-palette",
+                "spec": "https://w3c.github.io/csswg-drafts/css-fonts/#font-palette-prop",
+                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-fonts",
                 "countsTowardScore": true
             },
             "interop-2023-forms": {
                 "description": "Forms",
-                "mdn": "",
-                "spec": "",
-                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-forms", 
+                "mdn": "https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form",
+                "spec": "https://html.spec.whatwg.org/multipage/forms.html#the-form-element",
```

Comment:
This one is tricky, the 2022 forms category was general things about form controls, but the 2023 form category is about vertical form controls. So for 2023 spec, the CSS writing modes spec: https://drafts.csswg.org/css-writing-modes/#block-flow ,  more precisely this example https://drafts.csswg.org/css-writing-modes/#example-04885d8e seem to be more appropriate links).

Since the dashboard will display 2022/2023 combined, I think https://developer.mozilla.org/docs/Learn/Forms is a better link generally. The 2022 area covered everything from submission to styling form controls, so more than just the form element itself.

(I do notice that we've used the links you chose for 2022 already, so I guess it doesn't matter too much)